### PR TITLE
Upload file

### PR DESCRIPTION
**Pull request by N4tty93**

### CODE REVIEW

File: apps/kol-amelamdim/pages/api/upload-file.ts [hunk 1]
```diff
@@ -31,41 +35,22 @@ export default async function handler(req, res) {
...
-      //   (err, data) => {
-      //     console.log(data);
-      //   }
-      // );
-
-      s3.upload(params, (err, data) => {
+      s3.upload(params, (err, _) => {
         if (err) {
-          res.status(400).json(API_ERRORS.uploadFileError);
+          res.status(400).send(API_ERRORS.uploadFileError);
```
Comment (davidDoskalovich07) [inline apps/kol-amelamdim/pages/api/upload-file.ts]: we can go with the convention of returning json, what do you say about it ?
`return res.status(400).json(API_ERRORS.uploadFileError);`

**Review by davidDoskalovich07 — APPROVED** [review]
Very nice man!

**Review by davidDoskalovich07 — APPROVED** [review]
Very nice man!

File: apps/kol-amelamdim/pages/api/upload-file.ts (hunk 1 above)
Comment (N4tty93) [reply apps/kol-amelamdim/pages/api/upload-file.ts]: Fixed man

**Pull request merged**